api_scrape.py

```diff
@@ -1,17 +1,31 @@
-from bs4 import BeautifulSoup
-import urllib.request
-
+import requests
```

Comment:
Well done getting this to work using the `requests` module!

Reply:
Thank you! That was really rough since I tried out all sorts of methods and modules, but this one seemed to work best